jujugui/static/gui/src/app/templates/storefront-search-row.partial

```diff
@@ -0,0 +1,113 @@
+<div class="{{#if nameWidth}}{{nameWidth}}{{else}}four-col{{/if}} charm-name__column">
+  <h3 class="list-block__list--item-title">
+    {{#compare docType 'charm'}}
+      <img src="{{#if icon}}{{icon}}{{else}}{{STATIC_URL}}img/icons/default-charm.svg{{/if}}" class="list-icons__image charm__logo" alt="{{displayName}} icon" />
+    {{/compare}}
+    {{displayName}}
+    {{#if commercial}}
+      <span class="commercial-flag"></span>
+    {{/if}}
+  </h3>
+  {{#if tags}}
+    <ul class="tag-list">
+      {{#each tags}}
+        <li class="tag-list--item"><a href="/q?tags={{this}}">{{this}}</a></li>
+      {{/each}}
+    </ul>
+  {{/if}}
+</div>
+<div class="{{#if seriesWidth}}{{seriesWidth}}{{else}}two-col{{/if}} series__column">
+  {{#if series}}
+    <ul class="tag-list tag-list--spaced">
+      {{#each series}}
+        <li class="tag-list--item"><a href="{{url}}">{{name}}</a></li>
+      {{/each}}
+    </ul>
+  {{else}}
+    {{!Add some content so the row is spaced correctly.}}
+    &nbsp;
+  {{/if}}
+</div>
+<div class="{{#if iconsWidth}}{{iconsWidth}}{{else}}three-col{{/if}} charm-logos__column list-block__column">
+  <ul class="list-icons clearfix">
+    {{#each services}}
+      <li class="list-icons__item">
+        <a href="{{charm.url}}">
+          {{#compare 1 20 operator='>'}}
```

Comment:
Yup. Hack until we get on the newer version of Handlebars and can pass variables into the partial. The 1 will be replaced by one of those variables.